### ISSUE TITLE
Something is wrong with the output level of Shape

### ISSUE THREAD

**Issue by squinkylabs** [heuermh/VCVRack-Plugins#2]
It's putting out way more than the standard +/- 10
![image](https://user-images.githubusercontent.com/16314838/95148029-4197df80-0737-11eb-8bed-c9c7fd4abb67.png)

**Comment by heuermh** [comment]
Yeah it is!  As I mentioned in #1, I am still trying to figure out how to make clamp/rescaling work appropriately.  Or perhaps this never worked well in ChucK either, there isn't a nice oscilloscope like in VCV Rack.

**Comment by squinkylabs** [comment]
There are some really nice diagnostic tools in Rack. I really like the Submarine storage scope module.
